.github/actions/setup-rust/action.yml

```diff
@@ -0,0 +1,91 @@
+name: Set Up Rust
+description: Install the Rust toolchain, components, and set up the cache.
+inputs:
+  toolchain:
+    description: Type of toolchain to install.
+    type: string
+    required: false
+    default: stable
+  target:
+    description: Target triple of the toolchain.
+    type: string
+    required: false
+    default:
```

Comment:
Is this needed if it's just empty?

Reply:
This is just an explicit empty default, actually redundant.